cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -108,13 +110,6 @@ def filtered_list(cls):
         if not XBlockStudioConfigurationFlag.is_enabled():
             filtered_list.append('allow_unsupported_xblocks')
 
-        # TODO: https://openedx.atlassian.net/browse/EDUCATOR-736
-        # Before we roll out the auto-certs feature, move this to a good, shared
-        # place such that we're not repeating code found in LMS.
-        switches = WaffleSwitchNamespace(name=u'certificates', log_prefix=u'Certificates: ')
-        if not switches.is_enabled(u'instructor_paced_only'):
-            filtered_list.append('certificate_available_date')
```

Comment:
FYI @iloveagent57 I moved the `certificate_available_date` to the list on top - since it's NEVER shown on the course's Advanced Settings page - only in the Course's Date Settings page.

Reply:
👍